cli/cmd/initnodes.go

```diff
@@ -398,3 +418,35 @@ func gethInit(ctx context.Context, cfg InitConfig, dir string) error {
 
 	return nil
 }
+
+func downloadAndRestoreLatestSnapshot(ctx context.Context, network string, clientName string) error {
+	snapshotArchive := getSnapshotBackupArchive(clientName)
+	bucketName := fmt.Sprintf("omni-%s-snapshots", network)
+	gcpCloudStorageURL := fmt.Sprintf("https://storage.googleapis.com/%s/%s", bucketName, snapshotArchive)
+
+	log.Info(ctx, "Downloading and restoring latest snapshot...", "network", network, "client", clientName)
```

Comment:
providing the network isn't very useful, the user provided the network flag. Rather include the url.